apps/nextjs/src/components/AppComponents/common/ShareButtonGroup.tsx

```diff
@@ -0,0 +1,66 @@
+import { Box } from "@radix-ui/themes";
```

Comment:
Typo in the filename

Reply:
Is this a moved file?

Reply:
Yes

Reply:
It's just showing as an addition in the diff

Reply:
I've removed and readded using git mv. Should be correct now